images/atomic-openshift-descheduler.yml

```diff
@@ -1,10 +1,10 @@
 container_yaml:
   go:
     modules:
-    - module: github.com/kubernetes-incubator/descheduler
+    - module: sigs.k8s.io/descheduler
 content:
   source:
-    dockerfile: images/descheduler/Dockerfile.rhel7
+    dockerfile: Dockerfile
```

Comment:
`/Dockerfile` [1] uses `golang:1.13.5` as the base image. Which is not the same as  `registry.svc.ci.openshift.org/ocp/builder:golang-1.13` used in [2]. Also [2] is more safer as it's multi-stage build. Also, it seems Dockerfiles under `master/images/descheduler` are used for OpenShift, `/Dockerfile` for upstream. 

[1] https://github.com/openshift/descheduler/blob/master/Dockerfile#L14
[2] https://github.com/openshift/descheduler/blob/master/images/descheduler/Dockerfile.rhel7#L1

Reply:
@ingvagabund good catch, updated. Thanks